test/task_bulk_complete_test.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2012-2015 Spotify AB
```

Comment:
Feel free to change to your company, and set 2016 (I think that is how licenses should work, but I'm not sure :))